sympy/geometry/ellipse.py

```diff
@@ -151,6 +153,14 @@ def __new__(
         if hradius == 0 or vradius == 0:
             return Segment(Point(center[0] - hradius, center[1] - vradius), Point(center[0] + hradius, center[1] + vradius))
 
+        if (simplify(hradius).is_complex is True and simplify(hradius).is_real is False) or (simplify(vradius).is_complex is True and simplify(vradius).is_real is False):
+            raise GeometryError("Eccentricity of ellipse/circle should lie between [0, 1)")
+
+        if simplify(hradius).is_real and hradius !=0 and vradius == 0:
```

Comment:
I would not use simplify for these cases

Reply:
Done